src/docs_website/src/file_checker.zig

```diff
@@ -6,9 +6,15 @@ const assert = std.debug.assert;
 
 pub const file_size_max = 900 * 1024;
 
+// If this is set to true, we check if we get a 200 response for any external links.
+const check_links_external: bool = false;
```

Comment:
@matklad do you want this to be a command line option? Or is editing the source fine?

This is probably too flaky for the CI and takes too long. I guess we want to run this locally on occasion.

Reply:
Being in source is fine! We should _not_ run this on CI, as CI is forbidden from being flaky!

But there's value in checking it every once in a while!